src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+.conteiner {
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  max-width: $calendsar;
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  gap: $gap;
+  &__cell {
```

just for the sake of readability
```suggestion
  gap: $gap;

  &__cell {
```